src/game.cpp

```diff
@@ -493,7 +541,11 @@ ull Game::setBoardSize() {
   constexpr auto no_save_found_text =
       "No saved game found. Starting a new game.";
   constexpr auto board_size_prompt_text =
-      "Enter gameboard size (NOTE: Scores and statistics will be saved only for the 4x4 gameboard): ";
+
+      "Enter gameboard size (3x3 to 10x10). If you want to go back, enter '0': ";
+
+      "Enter gameboard size (3x3 to 10x10). If you want to go back, enter '0' (NOTE: Scores and statistics will be saved only for the 4x4 gameboard): ";
```

Comment:
Code duplication. 
When dealing with merge-conflicts, one must watch carefully for this.
Delete / Edit code which is duplicated in merge-conflicts.